shopinvader_schema_sale/schemas/__init__.py

```diff
@@ -0,0 +1,5 @@
+from . import shipping
```

Comment:
IMO, a good practice is to always import defined models a module root. It eases code readability....

```python

from odoo.addons.shopinvader_schema_sale.schemas import InvoicingInfo
```

Reply:
thanks for the tips